.cirrus.yml

```
@@ -20,10 +20,10 @@ env:
   ### Project variables
   DEPLOY_PULL_REQUEST: true
   ARTIFACTS: "\
-    org.sonarsource.scanner.commons:sonar-scanner-commons:jar,\
-    org.sonarsource.scanner.commons:sonar-scanner-commons-batch:jar,\
-    org.sonarsource.scanner.commons:sonar-scanner-commons-batch-interface:jar,\
-    org.sonarsource.scanner.commons:sonar-scanner-commons-parent:jar"
+    org.sonarsource.scanner.lib:sonar-scanner-java-library:jar,\
```

```suggestion
    org.sonarsource.scanner.jlib:sonar-scanner-java-library:jar,\
```
maybe?
To be consistent with the JIRA key?

---

I don't have a strong opinion but what kind of confusion are you thinking of? The goupId and package name are purely part of the java ecosystem so it might be a bit redundant.

---

Yeah, I always consider Maven more generic than Java since we tend to publish all kinds of artifacts on our repository.

Same, not a strong opinion :) maybe ask @claire-villard-sonarsource if she has a preference.

---

Not a strong opinion too neither :D 
I prefer `org.sonarsource.scanner.lib` because to me, adding `java` or `j` is redundant.
furthermore, the artifact ID already contains `-java-` to make it explicit in case we would have another lib for another language, and in that case, it could even be good to have them under the same group ID?

So in short, my vote goes to `org.sonarsource.scanner.lib`